client/src/components/SetUpComponent/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from "react";
+import axios from "axios";
+import API from "";
+
+const SetUpComponent = () => {
+  let categories = API.getCategories();
+  let gameCode = API.getQuizCode();
+  console.log("Make a quiz: ", categories, gameCode);
+
+  return (
+    <div>
+      <h2>Set up your game!</h2>
+      <form>
```

Comment:
@sjmarsnc we'll probably want a form action handler here to process the submit. Something like:

```suggestion
      <form onSubmit={(event) => handleSubmit(event)}>
```

with a `const handleSubmit = event => {}` before the return.